packages/server/src/data/classes/filter.js

```diff
@@ -0,0 +1,34 @@
+const _ = require('lodash');
+
+class Filter {
```

Comment:
For a lot of these, I’d like comments on the top describing what they are for. It’s not obvious to be from a quick glance what this is filtering.

Reply:
Yes, thanks will add some comments, it has a sense.

Reply:
Have added comments.